funnel/templates/proposal.html.jinja2

```diff
@@ -121,9 +121,9 @@
           <iframe class="facebooklike" src="//www.facebook.com/plugins/like.php?app_id=114496105304651&amp;href={{ request.url }}&amp;send=false&amp;layout=button_count&amp;width=150&amp;show_faces=false&amp;action=recommend&amp;colorscheme=light&amp;font&amp;height=21" scrolling="no" frameborder="0" style="border:none; overflow:hidden; width:150px; height:21px;" allowTransparency="true"></iframe>
         </div>
         <div class="col-md-3 col-lg-3 col-sm-12 col-xs-12">
-          Status: 
+          Status:
           {%- if 'view-status' in g.permissions %}
-            {{ PROPOSALSTATUS[proposal.status] }}
+            {{ PROPOSALSTATUS[proposal.status.value] }}
```

Comment:
You should no longer have `PROPOSALSTATUS` available in templates. Remove it from wherever this template is rendered. Change this to `{{ proposal.state.label }}`.